Context.sublime-menu

```diff
@@ -5,35 +5,35 @@
         [
```

Comment:
I noticed you added a `...` suffix to each command.
I noted that that that the ST devs add `...` in the context menu only to commands that open a new application, or switch focus to a different application.

See example:
- all commands that open Sublime Merge end with `...`
- open containing folder will open up the Files app
- open Terminal here, will open up the terminal :)
- I will stop with the examples 
![Uploading image.png…]()